ios/Encryption/AES.swift

```diff
@@ -0,0 +1,122 @@
+import Foundation
+import CommonCrypto
+
+struct AES {
+    private let key: Data
+    private let ivSize: Int         = kCCBlockSizeAES128
+    private let options: CCOptions  = CCOptions(kCCOptionPKCS7Padding)
+
+    enum Error: Swift.Error {
+        case invalidKeySize
+        case encryptionFailed
+        case decryptionFailed
+        case dataToStringFailed
+    }
+    
+    init(key: Data) throws {
+        guard key.count == EncryptionUtil.KEY_SIZE else {
+          throw Error.invalidKeySize
+        }
+        self.key = key
+    }
+}
+
+extension AES {
+    func encrypt(_ plainText: String) throws -> Data {
+        let dataToEncrypt = Data(plainText.utf8)
+        let bufferSize: Int = ivSize + dataToEncrypt.count + kCCBlockSizeAES128 // allow result round up to up the nearest multiple of 16 bytes
+        var buffer = Data(count: bufferSize)
+        try EncryptionUtil.randomData(length: ivSize, for: &buffer) //generate iv
```

Comment:
Just for consistency, could you please add a whitespace after `//` like all your other comments?